imap_processing/swe/l1b/swe_l1b_science.py

```diff
@@ -134,13 +135,45 @@ def convert_counts_to_rate(data: np.ndarray, acq_duration: int) -> npt.NDArray:
     return count_rate.astype(np.float64)
 
 
-def calculate_calibration_factor(time: int) -> None:
+def read_in_flight_cal_data() -> pd.DataFrame:
+    """
+    Read in-flight calibration data.
+
+    In-flight calibration data file will contain rows where each line
+    has 8 numbers, with the first being a time stamp in MET, and the next
+    7 being the factors for the 7 detectors.
+
+    This file will be updated weekly with new calibration data. In other
+    words, one line of data will be added each week to the existing file.
+    File will be in CSV format. Processing won't be kicked off until there
+    is in-flight calibration data that covers science data. TODO: decide
+    filename convention given this information. This function is a
+    placeholder for reading in the calibration data until we decide on
+    how to read calibration data through dependencies list.
+
+    Returns
+    -------
+    in_flight_cal_df : pandas.DataFrame
+        DataFrame with in-flight calibration data.
+    """
+    # TODO: Read in in-flight calibration file.
+
+    # Define the column headers
+    columns = ["met_time", "cem1", "cem2", "cem3", "cem4", "cem5", "cem6", "cem7"]
+
+    # Create an empty DataFrame with the specified columns
+    empty_df = pd.DataFrame(columns=columns)
+    return empty_df
+
+
+def calculate_calibration_factor(acquisition_time: float) -> npt.NDArray:
     """
     Calculate calibration factor.
 
     Steps to calculate calibration factor:
 
     1. Convert input time to match time format in the calibration data file.
+        Both input time and calibration data time should be in S/C MET.
```

Comment:
Rather than input_time do you want to use acquisition_time since that is the variable name input to this function?
```suggestion
    1. Convert the input time (acquisition_time) to match the time format in the calibration data file.
       Both the input time and calibration data time should be in S/C MET.
```